lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -597,6 +597,10 @@
     <datatype extension="bed_tabix.gz" type="galaxy.datatypes.interval:BedTabix" mimetype="application/octet-stream" display_in_upload="true"/>
     <datatype extension="gff_tabix.gz" type="galaxy.datatypes.interval:GffTabix" mimetype="application/octet-stream" display_in_upload="true"/>
     <datatype extension="bgzip" type="galaxy.datatypes.binary:CompressedArchive" subclass="true" display_in_upload="true"/>
+    <!-- GWAS results datatypes to be used with locuszoom visualization plugin -->
```

Comment:
Thanks for adding this comment and the time you put into all this :) This will be great!